core/src/main/java/ai/timefold/solver/core/impl/heuristic/selector/move/generic/list/ruin/ListRuinRecreateMove.java

```diff
@@ -87,7 +87,6 @@ protected void doMoveOnGenuineVariables(ScoreDirector<Solution_> scoreDirector)
 
             var nestedSolverScope = new SolverScope<Solution_>(solverScope.getClock());
             nestedSolverScope.setSolver(solverScope.getSolver());
```

Comment:
Why are we not passing the value range manager? This may cause the R&R to recalculate the unique list of values for entity ranges with every move.

Reply:
During my previous validation, I found a bug where the unique list of elements was always recalculated, causing a performance regression.

Reply:
At this point in the review, I understand that.